scripts/prepare_compliance_data.py

```diff
@@ -8,19 +8,38 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-import shutil
 import argparse
 import os
+import shutil
 import sys
 import glob
 import pysam
 import utils
+import requests
+import tempfile
+import zipfile
 
 
 class ComplianceDataMunger(object):
     def __init__(self, args):
         self.inputDirectory = args.inputDirectory
         self.outputDirectory = args.outputDirectory
+        # If no input directory is specified download from GitHub
+        if not args.inputDirectory:
+            print("Downloading test data...")
```

Comment:
I have been using utils.log for statements like this instead of print, which for now just calls print, but the advantage is that we can easily change the stream in the future if we decide to do that.